src/components/Borrow/BorrowTable/BorrowTable.tsx

```diff
@@ -24,10 +24,10 @@ interface IProp {
   currentDebtSign: string
   deposited: number
   depositedSign: string
-  cRatio: number
-  interestRate: number
-  liquidationPrice: number
-  maxBorrow: number
+  cRatio: string
```

Comment:
why did you this change?

Reply:
because printBN return string so I didn't change the type anymore